lib/idempotency.rb

```diff
@@ -11,6 +11,11 @@ class Idempotency
 
   setting :redis_pool
   setting :logger
+  setting :metrics do
+    setting :datadog_client
```

Comment:
Could we name it `statsd`?

```suggestion
    setting :statsd_client
```